CONTRIBUTING.md

```diff
@@ -215,12 +215,23 @@ When developing new features it can be helpful to enable logging. By setting `CD
 
 #### Terraform CDK
 
-1. Create a new branch (e.g. `prepare-release-0.5.0`)
+1. Create a new branch (e.g. `prepare-release-0.9.0`)
 2. Update the [CHANGELOG](./CHANGELOG.md)
 3. Update the version in the root `package.json`
-4. Create a PR to merge the new branch into `main`
-5. Merge the PR
-6. A new release will be build and published because the version changed
+4. Write an [upgrade guide](website/docs/cdktf/release/)
+5. Create a PR to merge the new branch into `main`
+6. Merge the PR
+7. A new release will be build and published because the version changed
+8. Update the prebuilt provider repository [like this](https://github.com/hashicorp/cdktf-repository-manager/pull/48) (The commit message need to have a `!` so that the minor version is bumped)
```

Comment:
```suggestion
8. Update the prebuilt provider repository [like this](https://github.com/hashicorp/cdktf-repository-manager/pull/48) (If the release contains breaking changes the commit message need to have a `!` after the scope so that the minor version is bumped. Example: `chore!: update cdktf version`)
```